website/events/management/commands/upload_gdrive_invoices.py

```diff
@@ -0,0 +1,63 @@
+import shutil
+import tempfile
+import os
+import os.path
+
+from django.core.management.base import BaseCommand
+from django.core.files.storage import default_storage
+
+from events.models import Expense
+from website.utils import gdrive
+
+# https://drive.google.com/drive/u/0/folders/1Kvfmz1eTNd9y2ZAqosaN3Cn2ydUKjtN_
+BASE_FOLDER = '1Kvfmz1eTNd9y2ZAqosaN3Cn2ydUKjtN_'
+
+
+class Command(BaseCommand):
+    help = "Generate the missing invoices"
+
+    def add_arguments(self, parser):
+        parser.add_argument('year', type=int)
+        parser.add_argument('month', type=int)
+
+    def handle(self, *args, **options):
+        year = int(options['year'])
+        month = int(options['month'])
+
+        expenses = Expense.objects.filter(
+            invoice_type=Expense.INVOICE_TYPE_A,
+            invoice_date__year=year,
+            invoice_date__month=month,
+        ).all()
+        print("Found {} expenses".format(len(expenses)))
+
+        for exp in expenses:
+            # build useful vars for later
+            orig_name = os.path.basename(exp.invoice.name)
+            extension = orig_name.rsplit('.')[-1].lower()
+            dest_filename = "{:%Y%m%d} - {}: {} [${}] ({}).{}".format(
+                exp.invoice_date, exp.event.name, exp.description,
+                exp.amount, orig_name, extension)
+            dest_foldername = exp.invoice_date.strftime("%Y%m")
+            print("Processing", repr(dest_filename))
```

Comment:
Es algo menor pero django recomienda usar `self.stdout.write()` o `self.stderr.write()` para imprimir en los comandos. 
https://docs.djangoproject.com/en/2.2/howto/custom-management-commands/

Reply:
"By using these proxies, it becomes much easier to test your custom command." 

Ojalá tuviera tests :)

Reply:
Gracias por el tip, though